tests/profiles-disabled/test_shipment_tags.py

```diff
@@ -0,0 +1,62 @@
+#  Copyright 2020 ShipChain, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+from rest_framework.reverse import reverse
+from shipchain_common.utils import random_id
+
+
+USER_ID = random_id()
+
+
+@pytest.fixture
+def shipment_tag_creation_data():
+    return {
+        'tag_type': '*Export-Shipment*',
+        'tag_value': '-->Europe',
+        'owner_id': USER_ID,
+    }
+
+@pytest.fixture
+def shipment_tag_creation_missing_owner_id():
+    return {
+        'tag_type': 'Destination',
+        'tag_value': 'Toronto'
+    }
+
+@pytest.fixture
+def entity_shipment_relationship(json_asserter, shipment):
```

Comment:
Did you intentionally redefine this? Fixtures in conftest don't need to be redefined.

Reply:
Oh this is a mistake it wasn't intentional to duplicate this.